src/redprl/refiner.fun

```diff
@@ -91,6 +91,21 @@ struct
           raise Fail "Did not match"
       end
 
+    fun FromLESS z _ jdg =
+      let
+        val _ = RedPrlLog.trace "Synth.FromLESS"
+        val (I, H) >> CJ.SYNTH tm = jdg
+        val Syn.INT = Syn.out tm
+        val CJ.LESS (a, b) = lookupHyp H z
+      in
+        if Abt.eq (a, tm) orelse Abt.eq (b, tm) then
+          T.empty #> (I, H, Syn.into Syn.INT)
+        else
+          raise Fail "Did not match"
+      end
+
+    fun FromWfHyp z = FromEQ z orelse_ FromLESS z
```

Comment:
I am confused by your comment so I will just say what is meant by "well-formedness goal" in the Nuprl tradition: it means a goal to establish that something is a member of some type. So, the term 'wellformedness' here made sense when the hypothesis was an equality judgment (it was a 'wellformedness hypothesis'), but doesn't make sense when the hypothesis is a `less-than` judgment or whatever.